### ISSUE TITLE
generate_custom_slots.py remove to many characters

### ISSUE THREAD

**Issue by stefanbode** [m0ngr31/kodi-alexa#113]
I do respect, that the script clean up some nasty characters that are in the names. Anyhow it kills ALL non aciii characters. Therefore in German, the very important üöä are also deleted. This is not acceptable because the Alexa skill can work with it and UTF8 should allow also. I assume any non-English language will have the same problem.

First occurrence line 16-19

**Comment by m0ngr31** [comment]
Are they being completely removed or are they getting replaced by `uoa`?

**Comment by stefanbode** [comment]
Hi, they are removed completely. The right behavior should be that only invalid chars get removed. Ouch it comes to French, Danish these are even more important. It will not work to use ascii only

Sent from remote phone

Am 19.02.2017 um 02:48 schrieb Joe Ipson <notifications@github.com<mailto:notifications@github.com>>:


Are they being completely removed or are they getting replaced by uoa?

**Comment by m0ngr31** [comment]
Are you trying the python version or the web version?

On the python version, I'm getting this:
```
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import kodi
>>> print kodi.sanitize_name(u'üöä')
uoa
```

**Comment by jingai** [comment]
Why are we stripping umlauts anyway?  Surely they're accepted by Amazon considering they allow German now?

**Comment by jingai** [comment]
It seems we might want to strip umlauts only if LANGUAGE=en.  Not sure, but I think we probably shouldn't be stripping them from source languages where they're actually necessary.

**Comment by jingai** [comment]
English skills allow unicode now as well, so we shouldn't strip them at all, IMHO.

**Comment by jingai** [comment]
This is fixed with #165.  After release, I'll make a note on the forums that users should re-generate their slot items.
